firmware/application/apps/ui_settings.cpp

```diff
@@ -1099,9 +1102,12 @@ SettingsMenuView::SettingsMenuView(NavigationView& nav)
 }
 
 void SettingsMenuView::on_populate() {
-    if (pmem::show_gui_return_icon()) {
+    const bool return_icon = pmem::show_gui_return_icon();
+
+    if (return_icon) {
```

Comment:
this is to match the implement style that in other places

Reply:
okay

Reply:
Haha, okay. Let's keep it like this.